<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CFindFuncConstBaseDialog</class>
 <widget class="QDialog" name="CFindFuncConstBaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Find OpenGL Function or Constant</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="m_groupboxconstant">
     <property name="title">
      <string>Find Constant</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLineEdit" name="m_lineeditconstant"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_pushbuttonfindconstant">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="m_groupboxfunction">
     <property name="title">
      <string>Find Function</string>
     </property>
     <layout class="QHBoxLayout">
      <item>
       <widget class="QLineEdit" name="m_lineeditfunction"/>
      </item>
      <item>
       <widget class="QPushButton" name="m_pushbuttonfindfunction">
        <property name="text">
         <string>Find</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3ListView" name="m_listviewresults">
     <column>
      <property name="text">
       <string>Name</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Value</string>
      </property>
      <property name="clickable">
       <bool>true</bool>
      </property>
      <property name="resizable">
       <bool>true</bool>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer name="spacer1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>470</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="m_pushbuttonok">
       <property name="text">
        <string>&amp;OK</string>
       </property>
       <property name="shortcut">
        <string>Alt+O</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListView</class>
   <extends>Q3Frame</extends>
   <header>q3listview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>m_pushbuttonfindconstant</sender>
   <signal>clicked()</signal>
   <receiver>CFindFuncConstBaseDialog</receiver>
   <slot>pushbuttonfindconstant_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>532</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonfindfunction</sender>
   <signal>clicked()</signal>
   <receiver>CFindFuncConstBaseDialog</receiver>
   <slot>pushbuttonfindfunction_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>596</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CFindFuncConstBaseDialog</receiver>
   <slot>pushbuttonclear_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>418</x>
     <y>573</y>
    </hint>
    <hint type="destinationlabel">
     <x>317</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_pushbuttonok</sender>
   <signal>clicked()</signal>
   <receiver>CFindFuncConstBaseDialog</receiver>
   <slot>pushbuttonok_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>517</x>
     <y>568</y>
    </hint>
    <hint type="destinationlabel">
     <x>598</x>
     <y>576</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lineeditconstant</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CFindFuncConstBaseDialog</receiver>
   <slot>lineeditconstant_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>49</y>
    </hint>
    <hint type="destinationlabel">
     <x>8</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_lineeditfunction</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CFindFuncConstBaseDialog</receiver>
   <slot>pushbuttonfindfunction_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>pushbuttonclear_clicked()</slot>
  <slot>lineeditconstant_textChanged(QString)</slot>
  <slot>lineeditfunction_textChanged(QString)</slot>
  <slot>pushbuttonok_clicked()</slot>
  <slot>pushbuttonfindconstant_clicked()</slot>
  <slot>pushbuttonfindfunction_clicked()</slot>
 </slots>
</ui>
